itest/lnd_psbt_test.go

```diff
@@ -1919,3 +1921,103 @@ func testPsbtChanFundingWithUnstableUtxos(ht *lntest.HarnessTest) {
 	block = ht.MineBlocksAndAssertNumTxes(1, 1)[0]
 	ht.AssertTxInBlock(block, txHash)
 }
+
+// lnrpcOutpointToStr returns a string representation of an lnrpc.OutPoint.
+func lnrpcOutpointToStr(outpoint *lnrpc.OutPoint) string {
+	return fmt.Sprintf("%s:%d", outpoint.TxidStr, outpoint.OutputIndex)
+}
+
+// testFundPsbtCustomLock verifies that FundPsbt correctly locks inputs
+// using a custom lock ID and expiration time.
+func testFundPsbtCustomLock(ht *lntest.HarnessTest) {
+	alice := ht.NewNodeWithCoins("Alice", nil)
+
+	// Define a custom lock ID and a short expiration for testing.
+	customLockID := ht.Random32Bytes()
+	lockDurationSeconds := uint64(30)
+
+	ht.Logf("Using custom lock ID: %s with expiration: %d seconds",
+		hex.EncodeToString(customLockID), lockDurationSeconds)
```

Comment:
nit: use `%x` instead of `hex.EncodeToString`?

Reply:
Done.